Astronomy/Astronomy/Helper Files/FetchPhotoOperation.m

```diff
@@ -0,0 +1,85 @@
+//
+//  FetchPhotoOperation.m
+//  Astronomy
+//
+//  Created by Karen Rodriguez on 5/19/20.
+//  Copyright © 2020 Wyatt Harrell. All rights reserved.
+//
+
+#import "FetchPhotoOperation.h"
+#import "Astronomy-Swift.h"
+
+@interface FetchPhotoOperation()
+
+@property BOOL executing;
+@property BOOL finished;
```

Comment:
I would add property attributes to explicitly add a setter/getter and to match the base class exactly except for `readwrite`